pytest.ini

```diff
@@ -0,0 +1,9 @@
+[pytest]
+python_files = test_*.py
+python_functions = test_*
+filterwarnings =
+    ignore::DeprecationWarning:traitlets.*:
+    #ignore:Automatic coloring is only supported for 3d fields.*:UserWarning
+    ignore:The truth value of an empty array is ambiguous.*:DeprecationWarning
+    ignore:invalid value encountered in divide:RuntimeWarning
+    ignore:`ipykernel.pylab.backend_inline` is deprecated.*:DeprecationWarning
```

Comment:
- We don't want this file, instead everything should go into `pyproject.toml`
- I am not sure if all these exceptions should really be defined. We should only ignore warnings in third-party libraries that are not caused by us.